### PR TITLE
Update truncated files

### PR DESCRIPTION
**Pull request by tylertomlinson**
PR to update games_truncated CSV file to add more games and make tests more robust.
Specifically done to remove the local league stats within several tests.
Changed expected within unit tests as well as some integration tests.
Rake passes with SimpleCov covering 97% 
Rspec also runs with methods passing.

Also updated Stat_tracker to have end on method line, which reduces lines.

### CODE REVIEW

File: lib/league_stats.rb [hunk 1]
```diff
@@ -83,8 +83,8 @@ def worst_fans
 
 # Helper Methods
   def find_name(id)
-    team = @teams.find { |team| team.team_id == id }
-    team.teamname
+    team_id = @teams.find { |team| team.team_id == id }
+    team_id.teamname
```
Comment (alex-latham) [inline lib/league_stats.rb:87]: Very clear! Thanks for making this more specific.

File: lib/stat_tracker.rb [hunk 2]
```diff
@@ -12,8 +12,7 @@ class StatTracker
   include DataLoadable
 
   def self.from_csv(locations)
-    StatTracker.new(locations)
-  end
+    StatTracker.new(locations) end
```
Comment (alex-latham) [inline lib/stat_tracker.rb:15]: Awesome. This will save us at least 40 lines!

File: test/league_stats_test.rb [hunk 3]
```diff
@@ -136,36 +135,33 @@ def test_game_teams_stats_update_id
...
 
     assert_equal expected, @league_stats.away_id_defense_stats
   end
 
   def test_defense_helper_returns_correct_hash
-    expected = {5=>3, 26=>2, 17=>2, 30=>6, 12=>1, 18=>1, 15=>2, 3=>3, 7=>3, 22=>2, 10=>2, 53=>3, 25=>3}
+    expected = {6=>10, 3=>14, 5=>10, 16=>13, 17=>10, 8=>14, 9=>9}
 
     assert_equal expected, @league_stats.defense_helper
   end
```
Comment (alex-latham) [inline test/league_stats_test.rb:158]: Good job figuring out this helper method test

File: lib/season_stats.rb [hunk 4]
```diff
@@ -1,10 +1,30 @@
...
+  end
+  # def least_accurate_team(season_id)
+  #   # Name of the Team with the worst ratio of shots to goals for the season
+  # end
+
+
+#   def shot_accuracy_by_team_id(team_id)
+#     (total_goals_by_team_id(team_id).to_f/total_shots_by_team_id(team_id) * 100.0)
+#     .floor
+#   end
```
Comment (alex-latham) [inline lib/season_stats.rb]: Should the commented lines here come out?

Comment (tylertomlinson) [reply lib/season_stats.rb]: Its a WIP will remove though for master

File: lib/stat_tracker.rb [hunk 5]
```diff
@@ -22,98 +21,74 @@ def initialize(locations)
...
 
   def highest_scoring_home_team
-    @league_stats.highest_scoring_home_team
-  end
+    @league_stats.highest_scoring_home_team end
 
   def lowest_scoring_visitor
-    @league_stats.lowest_scoring_visitor
-  end
+    @league_stats.lowest_scoring_visitor end
```
Comment (Maxwell-Baird) [inline lib/stat_tracker.rb:75]: Love how you are saving us so many lines with this format

File: lib/league_stats.rb (hunk 1 above)
Comment (whitneykidd) [reply lib/league_stats.rb:87]: ☝️

**Pull request merged**